DTS-STN/Life-Journeys
# LJ-24  setting up the canada.ca footer

# Description
  This is just the footer from the canada.ca theme that I did for another project a while back 
  it has more than actually needed css, but not even as close as the 600Kb css files of the canada.ca website    

Footer, landmark, woodmark, deleted obsolete file and updated the global.css 

## Test Instructions

1. Pull the branch and run 
2. Footer should work correctly in full medium and small screens resolutions
3. there are no tests 

## Meets Definition of Done

- [ ] TODO: Add Criteria
